### ISSUE TITLE
[FEATURE_REQUEST] Onboarding screen

### ISSUE THREAD

**Issue by urmilshroff** [urmilshroff/muze#8]
**Describe your idea**
Add an onboarding screen to explain users how the app works

**Suggest a way to implement it**
[This package](https://pub.dev/packages/flutter_overboard) looks good

**Comment by lordvins226** [comment]
Hi. can you provide illustrations for this feature?

**Comment by urmilshroff** [comment]
> Hi. can you provide illustrations for this feature?

I don't have illustrations yet, but you can try implementing [this](https://github.com/MangirdasKazlauskas/flutter-design-challenges) after which I'll correct the details/info.

**Comment by lordvins226** [comment]
> > Hi. can you provide illustrations for this feature?
> 
> I don't have illustrations yet, but you can try implementing [this](https://github.com/MangirdasKazlauskas/flutter-design-challenges) after which I'll correct the details/info.

Okay. Good i will try it

**Comment by Samkit-shah** [comment]
Hello is this issue solved?
If not can i work in it?

**Comment by urmilshroff** [comment]
> Hello is this issue solved? If not can i work in it?

Yeah sure, go ahead. Come up with something cool and attach a UI demo in your PR, but note that this project isn't really being worked upon and may never see the light of day.
